subprojects/testing-jvm/src/main/java/org/gradle/api/tasks/testing/Test.java

```diff
@@ -424,6 +427,45 @@ public void setDebug(boolean enabled) {
         forkOptions.setDebug(enabled);
     }
 
+    /**
+     * Enables fail fast behavior causing the task will fail on the first failed test.
+     */
+    @Option(option = "fail-fast", description = "Stops test execution after the first failed test.")
+    public Test failFastOn(boolean ignored) {
```

Comment:
❌ Unnecessary additional methods. We only need the getter and setter.

Reply:
see reply to next comment